src/App.js

```diff
@@ -0,0 +1,83 @@
+import './App.css';
+import React, { useState , useRef } from 'react';
+import ChatMessageList from './chat/ChatMessageList'
+import { v4 as uuidv4 } from 'uuid';
+import axios from "axios";
+
+const BASE_URL = "http://localhost:8080"
```

Comment:
TODO: Not hardcode the front end, this is for local testing, which is ok for now.